maintenance/wikia/WikiFactoryVariables/unescapeWordmarkText.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/**
+* Maintenance script to unescape wordmark-text which was double escaped by mistake
+* This is one time use script
+* @usage
+* 	# this will process wordmark-text for wiki with ID 119:
+* 	migrateWikiWordmarks --dry-run --wiki 119 --verbose
+*   # or
+*   /usr/wikia/backend/bin/run_maintenance '--script=wikia/WikiFactoryVariables/unescapeWordmarkText.php --dry-run --verbose' --id=119
+*/
+
+ini_set( 'display_errors', 'stderr' );
+ini_set( 'error_reporting', E_NOTICE );
+
+require_once( dirname( __FILE__ ) . '/../../Maintenance.php' );
+
+use \Wikia\Logger\WikiaLogger;
+
+/**
+ * Class MigrateWikiWordmarks
+ */
+class MigrateWikiWordmarks extends Maintenance {
+
+	protected $dryRun  = false;
+	protected $verbose = false;
+	const KEY_NAME = 'wordmark-text';
+	const WIKI_FACTORY_VARIABLE = "wgOasisThemeSettings";
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Unescapes wordmark-text variable in wgOasisThemeSettings";
+		$this->addOption( 'dry-run', 'Dry run mode', false, false, 'd' );
+		$this->addOption( 'verbose', 'Show extra debugging output', false, false, 'v' );
+		$this->addOption( 'file', 'File of wiki ids', false, true, 'f' );
+	}
+
+	private function unescape( $value ) {
+		$cnt = 0;
+		do {
+			$cnt++;
+			$old_value = $value;
+			$value = Sanitizer::decodeCharReferences( $value );
+			$value = htmlspecialchars_decode( $value, ENT_QUOTES );
+		} while ( $value !== $old_value || $cnt >= 10 );
+
+		return $value;
+	}
+
+	public function execute() {
+		global $wgCityId, $wgMedusaHostPrefix;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$wgMedusaHostPrefix" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[project-id]: wikia:app